pkg/proxy/iptables/proxier.go

```diff
@@ -581,16 +584,36 @@ func (proxier *Proxier) OnEndpointSlicesSynced() {
 // OnNodeAdd is called whenever creation of new node object
 // is observed.
 func (proxier *Proxier) OnNodeAdd(node *v1.Node) {
+	// This can not happen, the informer is filtering the node by name
```

Comment:
Instead of a comment, you want to remove this code in another PR?

Reply:
I've created a separate handler to handle the node events, it is much cleaner and apply to both proxiers ... it also allows to grow independently of the impementation